zimports.py

```diff
@@ -168,6 +170,7 @@ def _write_source(
     import_gap_lines,
     imports_start_on,
     style,
+    max_line_length=10000,
```

Comment:
why is this 10000, and like before, is there a time that we would call _write_source() and *not* pass max_line_length ?   should be positional required arg